docs/reference/release-notes/release-notes.md

```diff
@@ -6,23 +6,21 @@ description: "Release notes for Camunda 8 and its components."
 
 Camunda 8 release notes include notable new and improved features, enhancements, and bug fixes. Release notes are separated by minor release pages and include alphas released during the development cycle.
 
-[8.5.0](/reference/release-notes/850.md)
+[8.6](/reference/release-notes/860.md)
+[8.5](/reference/release-notes/850.md)
 
 ## Changelogs
 
-Changelogs for Camunda 8, including alphas, are also available on [GitHub](https://github.com/camunda/camunda-platform/releases). This includes release assets for Zeebe, Operate, Tasklist, and Identity.
-
-The current changelog can be found on the [Camunda repository](https://github.com/camunda/camunda-platform/releases/latest) and [Camunda Helm repository](https://github.com/camunda/camunda-platform-helm/releases/latest).
-
-[Update guides](/self-managed/operational-guides/update-guide/introduction.md) include links to both changelogs and release blogs.
+Changelogs for Camunda 8, including alphas, are available on [GitHub](https://github.com/camunda/zeebe/releases) and [Camunda Helm repository](https://github.com/camunda/camunda-platform-helm/releases).
 
 Most releases and changelogs are available through the [Camunda Platform repo](https://github.com/camunda/camunda-platform), however, the following component release notes are available as linked below:
 
 - [Desktop Modeler](https://github.com/camunda/camunda-modeler/releases)
+- [Connectors](https://github.com/camunda/connectors/releases)
 
 Some components have changelogs that are not publicly available, including the following:
 
 - Web Modeler Self-Managed
-- Optimize
```

Comment:
Is this true? Where is the Optimize Changelog publically available (I know it will change with MonoRepo, but speaking about status quo)

Reply:
Hmm, it was true when I originally wrote this.

I see Optimize in some camunda-platform releases, but I don't see it as an asset. It's not part of [8.5.0](https://github.com/camunda/camunda-platform/releases/tag/8.5.0).

Reply:
@RomanJRW Is the Optimize changelog public for 8.5.0? Did I miss it somewhere else?

Reply:
There was some time where we didn't have release note generation since our migration from JIRA -> Github. We have since introduced this and our release process to:
- Create the release notes in the private repo
- Manually paste them into the camunda-platform releases

It is possible that our release process wasn't modified at that time, meaning it wouldn't have been added to the public notes. However, I can see that they were generated [here](https://github.com/camunda/camunda-optimize/releases/tag/3.13.0). @akeller - please let me know if you would like us to retroactively add the 8.5.0 or any other release notes to the public ones

Reply:
@RomanJRW Oh whoops, I forgot Optimize could have separate versioning.... No need to add anything retroactively. I will just add another changelog URL for 8.5.

For 8.6, will the Optimize notes be in 8.6 via the mono-repo, or still separate (3.x)?

Reply:
I did see Optimize in 8.6.0-alpha1, so I think we are good moving forward as it's included in the core changelog 🙌

Reply:
@akeller - currently they exist in the core changelog because we manually paste them from the private repo. From 8.6 onwards, I would hope/expect that they are included by default anyway and we can drop that manual step